### PR TITLE
Add predicate option to ajax select

### PR DESCRIPTION
**Pull request by ypresto**
Add `:predicate` options to `as: :search_select` or etc.
It is passed to ransack instead of default `_contains` predicate.
This enables prefix search by specifing `predicate: :start`..!

### CODE REVIEW

File: app/assets/javascripts/activeadmin_addons/inputs/selected-list.js [hunk 1]
```diff
@@ -14,15 +14,17 @@ $(function() {
     });
 
     $('.selected-list-input', container).each(function(i, el) {
-      var url = $(el).data('url');
-      var fields = $(el).data('fields');
-      var displayName = $(el).data('display-name');
-      var method = $(el).data('method');
-      var model = $(el).data('model');
+      var element = $(el)
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/selected-list.js]: Missing semicolon  semi

**Comment by ypresto** [comment]
Anyone could you review this..? Thanks! 😃

File: spec/features/inputs/search_select_input_spec.rb [hunk 2]
```diff
@@ -149,4 +149,24 @@
       expect_select2_result_text_to_eq(1, @category2.name)
     end
   end
+
+  context "with different predicate" do
```
Comment (ldlsegovia) [inline spec/features/inputs/search_select_input_spec.rb]: @ypresto could you add some specs for selected list and nested select too?

**Comment by ypresto** [comment]
Sorry for late, me too >_<
Thanks for adding specs and docs, and confirmed abdc7786a48fb759544230b2b0ce5b6e96eeca6b !

**Pull request closed without merging**